src/tribler-gui/tribler_gui/error_handler.py

```diff
@@ -0,0 +1,87 @@
+import logging
+import os
+import traceback
+
+from tribler_common.sentry_reporter.sentry_reporter import SentryReporter
+
+from tribler_gui.dialogs.feedbackdialog import FeedbackDialog
+from tribler_gui.event_request_manager import CoreConnectTimeoutError
+
+
+class ErrorHandler:
+    def __init__(self, tribler_windows):
```

Comment:
Probably singular `tribler_window` should be used here, as this is the reference to the Tribler main window